### DISCUSSION TITLE
Very frequent entries in logs despite not actively using the app

### DISCUSSION THREAD

**Discussion by p-jones** [TandoorRecipes/recipes#2636] [General]
Thanks for creating this excellent software!

I have a question about logs and how frequently Tandoor is "doing things" despite not actively using the app.

My Tandoor setup has three containers: 

**1) NGINX:**
This service is logging events every 30s.
`
2023-09-17 07:13:22
127.0.0.1 - - [17/Sep/2023:11:13:22 +0000] "GET / HTTP/1.1" 302 0 "-" "Wget" "-"
2023-09-17 07:13:22
127.0.0.1 - - [17/Sep/2023:11:13:22 +0000] "GET /search/ HTTP/1.1" 302 0 "-" "Wget" "-"
2023-09-17 07:13:22
127.0.0.1 - - [17/Sep/2023:11:13:22 +0000] "GET /accounts/login/?next=/search/ HTTP/1.1" [200 7843](tel:200 7843) "-" "Wget" "-"
`

**2) DB**
This service logs VERY frequent events, too. It seems to be an error message. Can you please help me understand the error and rectify it?
`
2023-09-17 06:30:27
2023-09-17 10:30:27.343 UTC [30186] FATAL:  role "recipesuser" does not exist
2023-09-17 06:30:37
2023-09-17 10:30:37.457 UTC [30194] FATAL:  role "recipesuser" does not exist
2023-09-17 06:30:47
2023-09-17 10:30:47.562 UTC [30203] FATAL:  role "recipesuser" does not exist
`
Note the app runs despite the fatal error above.

**3) Tandoor app**
Log entries approximately every 4-10s.

`
2023-09-17 07:15:52
172.24.0.4 - - [17/Sep/2023:07:15:52 -0400] "GET / HTTP/1.0" 302 0 "-" "Wget"
2023-09-17 07:15:52
172.24.0.4 - - [17/Sep/2023:07:15:52 -0400] "GET /search/ HTTP/1.0" 302 0 "-" "Wget"
2023-09-17 07:15:52
172.24.0.4 - - [17/Sep/2023:07:15:52 -0400] "GET /accounts/login/?next=/search/ HTTP/1.0" [200 7843](tel:200 7843) "-" "Wget"
`

I just want to know this is normal and is not taxing my system. Any help is much appreciated!

**Comment by smilerz** [comment]
Something on your network is initiating web requests.  None of that is being triggered by tandoor itself.
